smac/facade/func_facade.py

```diff
@@ -15,31 +16,36 @@
 __license__ = "3-clause BSD"
 
 
-def fmin_smac(func: callable,
-              x0: list,
-              bounds: list,
+def fmin_smac(func: typing.Callable,
+              x0: typing.List[float],
+              bounds: typing.List[typing.List[float]],
               maxfun: int=-1,
               rng: np.random.RandomState=None,
+              scenario_args: typing.Mapping[str,typing.Any]
               **kwargs):
     """ Minimize a function func using the SMAC algorithm.
     This function is a convenience wrapper for the SMAC class.
 
     Parameters
     ----------
-    func : callable f(x)
+    func : typing.Callable
         Function to minimize.
-    x0 : list
+    x0 : typing.List[float]
         Initial guess/default configuration.
-    bounds : list
+    bounds : typing.List[typing.List[float]]
         ``(min, max)`` pairs for each element in ``x``, defining the bound on
         that parameters.
     maxfun : int, optional
         Maximum number of function evaluations.
     rng : np.random.RandomState, optional
             Random number generator used by SMAC.
+    scenario_args: typing.Mapping[str,typing.Any]
+        Arguments passed to the scenario
+        See smac.scenario.scenario.Scenario
```

Comment:
I think you need to apply some special syntax to link to the class, for example like here: https://github.com/automl/SMAC3/blob/master/smac/facade/smac_facade.py#L78

Reply:
hmm.. I would like to avoid to link to github. I think docstrings should be independent of github.

Reply:
Done